journalbeat/plan.sh

```diff
@@ -23,7 +23,8 @@ do_download() {
   SYSTEMD_INCLUDE_PATH=$(pkg_path_for core/systemd)/include
   GOPATH="$(dirname "${HAB_CACHE_SRC_PATH}")"
   export GOPATH
-  CGO_CFLAGS="-I${SYSTEMD_INCLUDE_PATH}" go get github.com/elastic/beats/journalbeat
+  rm -rf "${GOPATH}/src/github.com/elastic/beats"
```

Comment:
Is there anything that would create this path?

Reply:
I believe this is for repeated builds in the same studio, without this it would error out.